### PR TITLE
Upstream Vore Merges

### PR DESCRIPTION
**Pull request by Hawk-v3**
Also grabbed the pending radiation fixes because we kinda want those.

Do not merge until Izac's changes to resleeving-death are merged into this batch.

### CODE REVIEW

**Comment by Hawk-v3** [comment]
Note to self, before merge allowed, comment out the vore AI law stuff.

File: maps/tether/tether_defines.dm [hunk 1]
```diff
@@ -102,6 +102,8 @@
 	spawnpoint_stayed = /datum/spawnpoint/cryo
 
 	meteor_strike_areas = list(/area/tether/surfacebase/outside/outside3)
+	drop_pod_allowed_z = list(Z_LEVEL_SURFACE_HIGH)
```
Comment (Hawk-v3) [inline maps/tether/tether_defines.dm]: @RadiantFlash7 There's a new variable we might want to add to Cryo defines.

File: maps/~map_system/maps.dm [hunk 2]
```diff
@@ -59,6 +59,7 @@ var/list/all_maps = list()
 	var/list/holomap_legend_x = list()
 	var/list/holomap_legend_y = list()
 	var/list/meteor_strike_areas	// VOREStation Edit - Areas meteor strikes may choose to hit.
+	var/list/drop_pod_allowed_z		// VOREStation Edit - Which z-levels to target with droppods?
```
Comment (Hawk-v3) [inline maps/~map_system/maps.dm]: @RadiantFlash7 And it's use here.

**Comment by RadiantFlash7** [comment]
I.. I don’t even, Why is that a var they added?
Droppods are admin spawn only, unless they added a syndicate droppod-transport?

File: maps/~map_system/maps.dm (hunk 2 above)
Comment (RadiantFlash7) [reply maps/~map_system/maps.dm]: I... I don’t even, Why is that a var they added?
Droppods are admin spawn only, unless they added a syndicate droppod-transport?

Comment (Hawk-v3) [reply maps/~map_system/maps.dm]: See Devchat. :D

**Pull request merged**